extensions/charts/charts.go

```diff
@@ -26,6 +26,7 @@ const (
 	serverURLSettingID = "server-url"
 	rancherChartsName  = "rancher-charts"
 	active             = "active"
+	metadataName       = "metadata.name="
```

Comment:
@anupama2501 I added the const, but I didn't want to update every file with it yet in case you didn't want me to do that